packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -63,43 +65,44 @@ class _ProductPageState extends State<ProductPage> {
     final ColorScheme colorScheme = themeData.colorScheme;
     final MaterialColor materialColor =
         SmoothTheme.getMaterialColor(themeProvider);
+
     return Scaffold(
       backgroundColor: SmoothTheme.getColor(
         colorScheme,
         materialColor,
         ColorDestination.SURFACE_BACKGROUND,
       ),
-      appBar: AppBar(
-        title: Text(getProductName(_product, appLocalizations)),
-        actions: <Widget>[
-          PopupMenuButton<ProductPageMenuItem>(
-            itemBuilder: (BuildContext context) =>
-                <PopupMenuEntry<ProductPageMenuItem>>[
-              PopupMenuItem<ProductPageMenuItem>(
-                value: ProductPageMenuItem.WEB,
-                child: Text(appLocalizations.label_web),
-              ),
-              PopupMenuItem<ProductPageMenuItem>(
-                value: ProductPageMenuItem.REFRESH,
-                child: Text(appLocalizations.label_refresh),
```

Comment:
These should somewhere still be able to access

Reply:
@M123-dev I will add that asap.